paper/paper.tex

```diff
@@ -33,29 +33,28 @@
 % Use mixed case type for the shortened title
 % Ensure shortened title does not cause an overfull hbox LaTeX error
 % See ASPmanual2010.pdf 2.1.4  and ManuscriptInstructions.pdf for more details
-\markboth{Pey-Lian Lim, Eric Jeschke}{stginga: Ginga Plugins for Data
+\markboth{Pey Lian Lim, Eric Jeschke}{stginga: Ginga Plugins for Data
   Analysis and Quality Assurance of HST and JWST Science Data}
 
 \begin{document}
 
 \title{stginga: Ginga Plugins for Data Analysis and Quality Assurance of
 HST and JWST Science Data}
 
-% Note the position of the comma between the author name and the 
+% Note the position of the comma between the author name and the
 % affiliation number.
 % Author names should be separated by commas.
 % The final author should be preceded by "and".
 % Affiliations should not be repeated across multiple \affil commands. If several
 % authors share an affiliation this should be in a single \affil which can then
 % be referenced for several author names.
 % See ManuscriptInstructions.pdf and ASPmanual2010.pdf 3.1.4 for more details
-\author{Pey-Lian~Lim, Eric~Jeschke}
+\author{Pey~Lian~Lim, Eric~Jeschke}
 \affil{Space~Telescope~Science~Institute,~National~Astronomical~Observatory~of~Japan}
 
 % This section is for ADS Processing.  There must be one line per author.
-\paperauthor{Sample~Author1}{Author1Email@email.edu}{ORCID_Or_Blank}{Author1 Institution}{Author1 Department}{City}{State/Province}{Postal Code}{Country}
-\paperauthor{Sample~Author2}{Author2Email@email.edu}{ORCID_Or_Blank}{Author2 Institution}{Author2 Department}{City}{State/Province}{Postal Code}{Country}
-\paperauthor{Sample~Author3}{Author3Email@email.edu}{ORCID_Or_Blank}{Author3 Institution}{Author3 Department}{City}{State/Province}{Postal Code}{Country}
+\paperauthor{Pey~Lian~Lim}{lim@stsci.edu}{0000-0003-0079-4114}{Space~Telescope~Science~Institute}{Data~Analysis~Tools~Branch}{Baltimore}{MD}{21218}{USA}
+\paperauthor{Eric~Jeschke}{eric@naoj.org}{}{National~Astronomical~Observatory~of~Japan}{}{Hilo}{HI}{96720}{USA}
```

Comment:
@ejeschke , please verify institution info and provide ORCID and/or department name, if applicable. Thanks!

Reply:
Do you want me to push the change to this PR?

Reply:
@pllim, maybe you can just add it, since it's not a big change. orcid: 0000-0001-9586-4482.  Institution and associated info is correct.

Reply:
You can push changes too if you want. Otherwise I'll add it tomorrow. No department info?

Reply:
You could put "Subaru Telescope" for department info.